### PR TITLE
Add Beans and Bridge files, fix typo in cheat checkout and add cheat checkout tutorial.

### PR DESCRIPTION
**Pull request by Rdzanuwu**

### CODE REVIEW

File: HTML/Beanboozled Again.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
+﻿<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+        <title>Beanboozled Again — Keep Talking and Nobody Explodes Module</title>
+        <meta name="viewport" content="initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="css/main.css">
+        <link rel="stylesheet" type="text/css" href="css/font.css">
```
Comment (sleepin-fang) [inline HTML/Beanboozled Again.html:9]: Font before normalize

File: HTML/Beanboozled Again.html [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        <script src="js/ktane-utils.js"></script>
+        <style>
+            table {
+                margin: 1em auto;
+            }
+			.page-bg-01 {
```
Comment (sleepin-fang) [inline HTML/Beanboozled Again.html:15]: Indentation

File: HTML/Beanboozled Again.html [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+            background-image: url("img/Beanboozled Again/page-bg-noise-01.png");
+            }
+			@media screen and (min-width: 900px) {
```
Comment (sleepin-fang) [inline HTML/Beanboozled Again.html:18]: Look above

File: HTML/Beanboozled Again.html [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+            body {
+                background: url('img/Beanboozled Again/web-background.jpg');
+
```
Comment (sleepin-fang) [inline HTML/Beanboozled Again.html:21]: To be removed.

**Comment by samfundev** [comment]
Merged in 846a8f18043bf6aacd3db090b3089edfbcffae7d

**Pull request closed without merging**